workflows/qc/quality-check-standard.wdl

```diff
@@ -58,9 +60,11 @@ workflow quality_check {
         bam: "Input BAM format file to quality check"
         bam_index: "BAM index file corresponding to the input BAM"
         reference_fasta: "Reference genome in FASTA format"
+        coverage_beds: "An array of 3 column BEDs which are passed to the `-b` flag of mosdepth, in order to restrict coverage analysis to select regions"
+        coverage_labels: "An array of equal length to `coverage_beds` which determines the prefix label applied to the output files. If omitted, defaults of `regions1`, `regions2`, etc. will be used."
         gtf: "GTF features file. **Required** for RNA-Seq data"
         star_log: "Log file generated by the RNA-Seq aligner STAR"
-        experiment: "'WGS', 'WES', or 'RNA-Seq'"
+        experiment: "'DNA' or 'RNA'"
```

Comment:
I don't have any useful thoughts on this. I agree that `experiment` is no longer truly correct. Maybe `molecule`? Or `nucleic_acid`?

Reply:
I like `molecule` the best, but don't love it. Maybe @claymcleod has thoughts on a better word?

Reply:
I like `molecule` myself. I'm good with this change in general.